### ISSUE TITLE
Training instabilities

### ISSUE THREAD

**Issue by ricshaw** [OpenRobotLab/AnySplat#68]
I am training with 4 nodes x 8 gpus, with the DL3DV dataset, and after about 1000 training steps I am encountering "Skipping batch with high loss (0.249768) at step 1001 on Rank 25". From this point onwards, the loss remains high and the model does not train and the outputs go wrong....

Early in training:

![Image](https://github.com/user-attachments/assets/d929d5fd-d2c6-475c-9672-11b7295e0624)

Later in training:

![Image](https://github.com/user-attachments/assets/0c08e544-f26d-4071-9b40-0dbbefec857c)

**Comment by myc634** [comment]
Hi, could you please provide more context to help us debugging? Like the loss and visualisation before this information on the log. In our expriment, this kind of "Skipping batch with high loss" happens when the input case is hard and model fail to give a correct prediction, so we will skip this training step to keep a stable training

**Comment by ricshaw** [comment]
I managed to avoid this issue by switching to precision "32-true". Now it trains fine on DL3DV. It seems precision: "16-mixed" was the problem....

**Comment by Alex417ii** [comment]
Is it by changing the precision parameter in the configuration file to "32" that this issue was resolved?
trainer:
  max_steps: 3000
  val_check_interval: 500
  num_nodes: 1
  accumulate_grad_batches: 1
  precision: "32"  # bf16-mixed

**Comment by JamChaos** [comment]
@Alex417ii  Hi！ Do you solved it?
